devutils/lib/bash/bazel.bash

```diff
@@ -39,7 +39,11 @@ Bazel::TestTargets () {
 
     while read target
```

Comment:
One should always use the `-r` option of `read` for escaping matters, unless having an explicit reason not to do so